lib/src/client_context.dart

```diff
@@ -24,18 +24,20 @@ abstract class ClientContext {
   Assets get assets;
 }
 
-abstract class Services {
-  DatastoreDB get db;
+class Services {
+  final DatastoreDB db;
 
-  /// Access the gcloud package storage API. This will autumatically be
+  /// Access the gcloud package storage API. This will automatically be
   /// available on deployed App Engine applications. For local testing this
-  /// will be `null` unless the environemnt variable
+  /// will be `null` unless the environment variable
   /// `STORAGE_SERVICE_ACCOUNT_FILE` is passed in `app.yaml`. This environment
   /// variable should point to a private key for a service account in JSON
   /// format.
```

Comment:
Remove this comment, it's out of date: this environment variable doesn't get used anymore.